src/ApiService/ApiService/Functions/Pool.cs

```diff
@@ -133,7 +133,8 @@ private static PoolGetResult PoolToPoolResponse(Service.Pool p)
                     HeartbeatQueue: queueSas,
                     InstanceId: instanceId,
                     ClientCredentials: null,
-                    MultiTenantDomain: _context.ServiceConfiguration.MultiTenantDomain)
+                    MultiTenantDomain: _context.ServiceConfiguration.MultiTenantDomain,
+                    IsUnmanaged: !p.Managed)
```

Comment:
I know managed is our current default which is why you added IsUnmanaged but I think it'll be easier to reason about in the future if we do `IsManaged`. Specifically, to avoid the double negative with `!IsUnmanaged`.